modules/ssh/src/main/java/org/apache/ignite/internal/util/nodestart/StartNodeCallableImpl.java

```diff
@@ -203,7 +203,7 @@ public StartNodeCallableImpl(IgniteRemoteStartSpecification spec, int timeout) {
 
             shell(ses, "mkdir " + scriptOutputDir);
 
-            String scriptOutputPath = scriptOutputDir + separator + scriptOutputFileName;
+            String scriptOutputPath = "C:\\Users\\user\\log.log" + separator + scriptOutputFileName;
```

Comment:
?

Reply:
something went wrong - part of dev commit of other ticket - reverted